### DISCUSSION TITLE
Is it possible to add a "reset statistics" option?

### DISCUSSION THREAD

**Discussion by amiteshore** [monkeytypegame/monkeytype#770] [Ideas] [answered]
My average wpm on QWERTY layout is 110 wpm, and I'm learning DVORAK (20 wpm).
Forgetfully, I practiced on both layouts and mixed up my avg. wpm. My avg. wpm reduced a lot. So is it possible to have an option like **reset statistics** or **different statistics for different keyboard layout**?

**Reset statistics** option on the account page is a nice option to have like keybr.com.

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
I don't know if you know this but you can use tags for separate wpm averages instead of using the main thing for everything you type. If you already know what tags are then since there isn't really a "reset statistics" option so you'd have to make a new tag for your QWERTY wpm and another tag for your DVORAK wpm if you want to have separate pb's and such. I think if you really want to you could also just make another account.
